<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QTreeView" name="parameter_view">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="SearchBox" name="parameter_search_box" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>CAN</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QTreeView" name="can_view">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="SearchBox" name="can_search_box" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Static Modbus</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QTreeView" name="static_modbus_view"/>
           </item>
           <item row="0" column="0">
            <widget class="SearchBox" name="static_modbus_search_box" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Value Set</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QTreeView" name="value_set_view"/>
           </item>
           <item row="0" column="0">
            <widget class="SearchBox" name="value_set_search_box" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>SunSpec1</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QTreeView" name="sunspec1_view"/>
           </item>
           <item row="0" column="0">
            <widget class="SearchBox" name="sunspec1_search_box" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>SunSpec2</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QTreeView" name="sunspec2_view"/>
           </item>
           <item row="0" column="0">
            <widget class="SearchBox" name="sunspec2_search_box" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget22">
       <layout class="QGridLayout" name="gridLayout22">
        <item row="2" column="0">
         <widget class="QTreeView" name="check_result_view">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="SearchBox" name="check_result_search_box" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="action_new_value_set_2">
     <property name="title">
      <string>&amp;Value Set</string>
     </property>
     <addaction name="action_new_value_set"/>
     <addaction name="action_open_value_set"/>
     <addaction name="action_save_value_set"/>
     <addaction name="action_save_as_value_set"/>
    </widget>
    <widget class="QMenu" name="menu_Project">
     <property name="title">
      <string>&amp;Project</string>
     </property>
     <addaction name="action_new_project"/>
     <addaction name="action_open_project"/>
     <addaction name="action_save_project"/>
     <addaction name="action_save_as_project"/>
    </widget>
    <addaction name="menu_Project"/>
    <addaction name="action_new_value_set_2"/>
    <addaction name="separator"/>
    <addaction name="action_import_smdx"/>
    <addaction name="action_full_export"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_check"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_project">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_save_as_project">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_save_project">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_new_project">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_import_sym">
   <property name="text">
    <string>&amp;Import .sym...</string>
   </property>
  </action>
  <action name="action_export_sym">
   <property name="text">
    <string>&amp;Export .sym...</string>
   </property>
  </action>
  <action name="action_Project">
   <property name="text">
    <string>&amp;Project</string>
   </property>
  </action>
  <action name="action_new_value_set">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Value_Set">
   <property name="text">
    <string>&amp;Value Set...</string>
   </property>
  </action>
  <action name="action_save_value_set">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_save_as_value_set">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_open_value_set">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_import_smdx">
   <property name="text">
    <string>Import &amp;SMDX...</string>
   </property>
  </action>
  <action name="action_full_import">
   <property name="text">
    <string>Full import...</string>
   </property>
  </action>
  <action name="action_full_export">
   <property name="text">
    <string>&amp;Full Export...</string>
   </property>
  </action>
  <action name="action_full_export_first_time">
   <property name="text">
    <string>Full Export (First Time)...</string>
   </property>
  </action>
  <action name="action_check">
   <property name="text">
    <string>&amp;Check</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchBox</class>
   <extends>QWidget</extends>
   <header>epyqlib.searchbox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
